pcsx2/GSDumpReplayer.cpp

```diff
@@ -276,10 +276,8 @@ void GSDumpReplayerCpuStep()
 			u32 size;
 			std::memcpy(&size, packet.data, sizeof(size));
 
-			std::unique_ptr<u8[]> arr(new u8[size]);
-			GetMTGS().WaitGS();
-			GetMTGS().SendPointerPacket(GS_RINGTYPE_INIT_READ_FIFO2, size, arr.get());
-			GetMTGS().WaitGS(false); // wait without reg sync
+			std::unique_ptr<u8[]> arr(new u8[size * 16]);
```

Comment:
I assume this was broken before?

Reply:
Yeah, but because it never actually did the read (only set it up/init'ed it), it never caused any issues.

Reply:
That's why readbacks didn't trigger in GSdumps isn't it

Reply:
They always seemed to trigger for me? It just never copied it to the DMA buffer (`arr` in this case).

Reply:
Really?  There are a number of games that do that for me
e.g. this VP2 trace had 1 readback during the game when it was taken, but 0 in trace playback: [VP2_MissingReadback.gs.xz.zip](https://github.com/PCSX2/pcsx2/files/8551468/VP2_MissingReadback.gs.xz.zip)

Doesn't look like this changes it though.  Must be for a different reason.

Reply:
Pretty strange, the init fifo should get recorded into the dump when it happens. I'll double-check that.